### PR TITLE
WED-13 : Create a Event table and Wedding-Event association table

### PR DESCRIPTION
**Pull request by rohanjogatar**

### CODE REVIEW

File: weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java [hunk 1]
```diff
@@ -38,6 +38,9 @@ public Response add(final String jsonString) {
 
         try {
             Wedding wedding = (Wedding) session.get(Wedding.class, weddingId);
+            if(wedding == null) {
```
Comment (neeljain198) [inline weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java:41]: space after if

File: weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java [hunk 2]
```diff
@@ -49,4 +52,14 @@ public Response add(final String jsonString) {
...
+
+    /**
+     * To retreive device tokens from wedding object -
+     *
+     session.createCriteria(DeviceToken.class).add(Restrictions.eq(
+     "Wedding",wedding
+     )).list()
+
+     */
+    //TODO try removing wedding.getdevicetokens() same from event.getWeddingEvents
```
Comment (neeljain198) [inline weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java:64]: what is this todo about?

File: weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java [hunk 3]
```diff
@@ -49,4 +52,14 @@ public Response add(final String jsonString) {
             session.close();
         }
     }
+
+    /**
```
Comment (neeljain198) [inline weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java:56]: do we need this?

File: weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java (hunk 2 above)
Comment (rohanjogatar) [reply weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java:64]: Currently the wedding object does not store device tokens associated to it. To get the device token you need to do something like -
`session.createCriteria(DeviceToken.class).add(Restrictions.eq(
     "Wedding",wedding
     )).list()`

I am unable to figure out why this is happening. Kept it as TODO as did not feel it to be a big hurdle now

File: weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java (hunk 3 above)
Comment (rohanjogatar) [reply weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java:56]: Can go off..

**Pull request merged**